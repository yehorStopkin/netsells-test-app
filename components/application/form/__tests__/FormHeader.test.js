const { shallowMount } = require('@vue/test-utils');
const FormHeader = require('@/components/application/form/FormHeader').default;

describe('FormHeader', () => {
    test('component default', () => {
        const wrapper = shallowMount(FormHeader, { propsData: { title: 'Test title' } });
        expect(wrapper.exists()).toBeTruthy();
        expect(wrapper.element).toMatchSnapshot();
    });
});
