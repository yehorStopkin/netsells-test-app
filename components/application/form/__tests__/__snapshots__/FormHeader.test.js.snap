// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormHeader component default 1`] = `
<div
  class="form-header"
>
  <div
    class="form-header__title"
  >
    
        Test title
    
  </div>
   
  <div
    class="form-header__subtitle"
  >
    <span
      class="form-header__subtitle required"
    >
       Required fields are marked by an asterisk 
    </span>
  </div>
</div>
`;
